pkg/controller/defaults/testdata/region/default/expected.golden

```diff
@@ -21,15 +21,13 @@ status:
   appStatus: {}
   columns: {}
   conditions:
-    reason: Success
-    status: "True"
-    success: true
+  - error: true
+    message: 'Unknown resource from test: *v1.ProjectInstance.Unknown group from test:
+      *v1.ProjectInstance "app-namespace" not found'
+    reason: Error
+    status: "False"
     type: defaults
-  defaults:
-    memory:
-      "": 0
-      container-name: 0
-    region: local
+  defaults: {}
```

Comment:
It looks like this and several other tests went from expecting success to expecting failure. Was this intended?

Reply:
I thought I fixed all of these, my apologies. I will update.